package.json

```diff
@@ -3,14 +3,17 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@incodetech/welcome": "1.69.1",
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1",
     "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "react-scripts": "5.0.1",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "HTTPS=true SSL_CRT_FILE=./.cert/cert.pem SSL_KEY_FILE=./.cert/key.pem react-scripts start",
+    "start": "HTTPS=true PORT=5173 HOST=0.0.0.0 react-scripts start",
```

Comment:
no need to add cert files
expose server to the local network (for mobile testing)